src/bandit-evaluator.spec.ts

```diff
@@ -297,4 +302,121 @@ describe('BanditEvaluator', () => {
       ).toBe(true);
     });
   });
+
+  describe('selectAction', () => {
+    const flagKey = 'flag';
+    const actionWeights = { action1: 0.2, action2: 0.5, action3: 0.3 };
+
+    it('selects actions', () => {
+      expect(exposedEvaluator.selectAction(flagKey, 'subjectA', actionWeights)).toBe('action1');
+      expect(exposedEvaluator.selectAction(flagKey, 'subjectB', actionWeights)).toBe('action2');
+      expect(exposedEvaluator.selectAction(flagKey, 'subjectE', actionWeights)).toBe('action3');
+    });
+  });
+
+  describe('evaluateBandit', () => {
+    it('evaluates the bandit with action contexts', () => {
```

Comment:
Based on [this Python SDK test](https://github.com/Eppo-exp/python-sdk/blob/main/test/bandit_test.py#L221), but instead of mocking out the sharder, we choose two subjects that hit each of the two actions.